### PR TITLE
Fixes /whitepaper route and adds helmet to whitepaper component

### PR DESCRIPTION
**Pull request by Phasor**

### CODE REVIEW

File: src/pages/Whitepaper/whitepaperContent.tsx [hunk 1]
```diff
@@ -5,8 +5,8 @@ const htmlContent = `<!DOCTYPE html>
     <!-- Required meta tags -->
```
Comment (piersss) [inline src/pages/Whitepaper/whitepaperContent.tsx:5]: I just noticed whitepaperContent has html body and head tags, they shouldn't be in there.

**Review by piersss — CHANGES_REQUESTED** [review]
Nice!

File: src/pages/Whitepaper/whitepaperContent.tsx (hunk 1 above)
Comment (Phasor) [reply src/pages/Whitepaper/whitepaperContent.tsx]: Good spot. Updated below.

File: src/pages/Whitepaper/whitepaperContent.tsx [hunk 2]
```diff
@@ -539,7 +502,6 @@ cancel(&lt;makerAddress&gt;, 5, 'GNO', &lt;takerAddress&gt;, 10, 'BAT', &lt;expi
       ga('create', 'UA-98809379-1', 'auto')
```
Comment (piersss) [inline src/pages/Whitepaper/whitepaperContent.tsx:502]: What about all these scripts? Are they necessary?

**Review by piersss — CHANGES_REQUESTED** [review]

**Comment by piersss** [comment]
I have added the commits to my branch, so I will close this PR.

**Pull request closed without merging**